formula/odin@1.2.0-alpha.rb

```diff
@@ -0,0 +1,34 @@
+# typed: false
+
+class OdinAT111Alpha < Formula
```

Comment:
Both files are identical after the change?

Reply:
only class name is different. Was referring https://github.com/dream11/homebrew-tools/pull/4/files